packages/azure/src/integration.spec.ts

```diff
@@ -14,7 +15,7 @@ test('Azure OpenAI Text Streaming', async () => {
     HashbrownAzure.stream.text({
       apiKey: AZURE_API_KEY,
       endpoint: AZURE_ENDPOINT,
-      request,
+      request: request as AzureCompletionCreateParams,
```

Comment:
Don't love having to use type assertion here, may end up abandoning this PR

Reply:
I think that is fine, especially in a test.